docs/coding-guidelines.md

```diff
@@ -192,7 +192,23 @@ For example the following are correct:
         return (string input) => // method body
     }
     ```
+1. Prefer a using directive over fully qualifying a type.
 
+    This is correct:
```

Comment:
https://github.com/NuGet/NuGet.Client/pull/3563#discussion_r465409325 😊

I just think it's funny. The length/verbosity doesn't bother me.

Reply:
I don't like it :( But I don't have a better suggestion.

Reply:
Maybe saying 
```suggestion
    ```cs
    using System.Threading.Tasks;
    Task DoSomethingAsync()
    ```
   instead of
    ```cs
    using System;
    System.Threading.Tasks.Task DoSomethingAsync()
    ```
```